ui/components-lib/src/components/ProposedChangesCard.scss

```
@@ -0,0 +1,168 @@
+    align-items: center;
+    flex-shrink: 0;
+
+    svg {
+      width: 16px;
+      height: 16px;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-family: 'Heebo', sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Heebo" (font-family-name-quotes)

(font-family-name-quotes)